src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -663,6 +663,27 @@ def test_directive_lastprivate(fortran_reader, fortran_writer):
                      "variables marked as private after the OpenMP region")
 
 
+def test_omp_parallel_private_clause():
+    '''Test the private_clause method in the OMPParallelDirective
+    class.
+
+    '''
+    omp_parallel = OMPParallelDirective.create()
+    clause = omp_parallel.private_clause
+    assert isinstance(clause, OMPPrivateClause)
+
+
+def test_omp_parallel_encloses_omp_directive():
+    '''Test the _encloses_omp_directive method in the OMPParallelDirective
+    class. This tests where there is no OMPRegionDirective within the
+    OMPParallelirective. Nothing is actually done or returned in the
+    method at the moment so this test is purely for coverage.
+
+    '''
+    omp_parallel = OMPParallelDirective.create()
+    omp_parallel._encloses_omp_directive()
+
+
```

Comment:
The method itself has nothing to do with this PR, so I don't think this PR should be removing a method that has been added to master from another PR. The only reason I added this test is because I was getting reduced coverage for the whole code (even though there was 100% coverage for the changes) and codecov reported this method as a culprit.